INSTALL.rst

```diff
@@ -0,0 +1,196 @@
+###########
+Cycamore Installation Guide
+###########
+
+Cycamore uses the CMake cross platform bailed-generator as a build system. To
+simplify the usage of cmake to build Cycamore from source, a python script is
+provided with the source.
+
+.. contents:: Table of Contents
+
+******************************
+Dependencies
+******************************
+
+A full list of the Cycamore package dependencies is shown below:
+
+====================   ==================
+Package                Minimum Version
+====================   ==================
+`Cyclus`               1.4
+====================   ==================
+
+You can find the Cyclus Core which is require to use the Cycamore additionnal
+Modelus. You can download Cyclus from the git repository.
+(`GitHub Cyclus Repository <https://github.com/cyclus/cyclus>`_):
+
+- Clone the Cyclus Repo: ``git clone
+  https://github.com/cyclus/cyclus.git``,
+
+- Install Cyclus  in ``~/.local/``:
+  ``python install.py`` from the Cyclus folder.
+
+We also provide installation instruction using Conda, Deb, as well as source
+installation from a tarball. Please wisit our `additionnal dependencies
+instruction <DEPENDENCIES.rst>`_.
+
+
+******************************
+Installation
+******************************
+
+Before going further on the installation procedure be sure you have installed
+all the required dependencies listed above.
+
+
+Default installation
+------------------------
+
+Run the install script:
+
+.. code-block:: bash
+  
+  python install.py
+
+
+If you successfully followed the instruction above Cycamore library have been
```

Comment:
If you successfully followed the instructions above, then the Cycamore library has been generated and placed in the . . . .